ansible/playbook/tasks/roles/sqlite-to-postgres/tasks/main.yml

```diff
@@ -0,0 +1,69 @@
+---
+- name: Create Grafana backup dir
+  file:
+    path: "{{ grafana_backup_dir }}"
+    state: directory
+    owner: grafana
+    group: grafana
+    mode: '0700'
+
+- name: Stop grafana before upgrade
+  supervisorctl:
+    name: 'grafana'
+    state: stopped
+
+- name: Create grafana user
+  postgresql_user:
+    db: grafana
+    name: grafana
+    password: grafana
+    priv: 'ALL'
+    expires: infinity
+    state: present
+  when: not ansible_check_mode
+
+- name: Create actual grafana config
+  include_role:
+    name: grafana
+
+- name: Start grafana again
+  supervisorctl:
+    name: grafana
+    state: started
+
+- name: wait grafana database initiazation
+  pause:
+    seconds: 10
+
+- name: Stop grafana before upgrade
+  supervisorctl:
+    name: 'grafana'
+    state: stopped
+
+- name: Install grafana-migrate
+  yum:
+    name: "grafana-db-migrator"
+    state: latest
+
+- name: Run grafana migrator
+  command: grafana-migrate /srv/grafana/grafana.db "postgres://grafana:grafana@localhost:5432/grafana?sslmode=disable"
```

Comment:
Maybe better call it `grafana-migrate-db` to make it obvious?

Reply:
Renamed